### PR TITLE
Add intro modal

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

File: src/components/application.js [hunk 1]
```diff
@@ -83,7 +97,12 @@ export default class Application extends Component {
             />
           </div>
         </div>
-        <Dock title={p.title}>
+        <Dock>
+          <div className='TauiTitle'>
+            <Icon icon='map' />
+            <span>{p.title || message('Title')}</span>
+            {p.info && <Icon icon='info-circle' onClick={this._showInfo} />}
```
Comment (ansoncfit) [inline src/components/application.js]: In Chrome on my machine, the cursor does not indicate this icon is clickable on hover. Is there a way to more clearly identify this info icon as a link, and maybe right-justify it to give the title more space?

File: src/components/map.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import useMarker from '../hooks/use-marker'
 import usePointsOfInterest from '../hooks/use-points-of-interest'
 
 import Icon from './icon'
+import MapClickControl from './map-click-control'
```
Comment (ansoncfit) [inline src/components/map.js:14]: 👍

File: yarn.lock [hunk 3]
```diff
@@ -1717,6 +1717,11 @@ array-differ@^1.0.0:
   resolved "https://registry.yarnpkg.com/array-differ/-/array-differ-1.0.0.tgz#eff52e3758249d33be402b8bb8e564bb2b5d4031"
   integrity sha1-7/UuN1gknTO+QCuLuOVkuytdQDE=
 
+array-differ@^2.0.3:
```
Comment (ansoncfit) [inline yarn.lock:1720]: Should this file be `.gitignore`d?

**Review by ansoncfit — APPROVED** [review]
A couple minor requests. Otherwise, looks good to go.

File: yarn.lock (hunk 3 above)
Comment (trevorgerhardt) [reply yarn.lock:1720]: Nope, it helps ensure the exact packages are installed for two different people.

File: src/components/application.js (hunk 1 above)
Comment (trevorgerhardt) [reply src/components/application.js]: Will do 👍

Comment (trevorgerhardt) [reply src/components/application.js]: Changed in https://github.com/conveyal/taui/pull/127/commits/32f454bfafe94eef970507b00f92cbc5d2439246

**Comment by trevorgerhardt** [comment]
Btw, @ansoncfit you can see the changes in PRs if you click on "show environments" click on the generated link out. Auto-deployments use an empty config, but you can copy the config from configurations/taui/... etc. The new configuration only has a single "store.json"

**Comment by trevorgerhardt** [comment]
I intend to get this working for Analysis...soon :)

**Pull request merged**